integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -141,6 +142,12 @@ private void cleanKafkaTopics() {
     private void cleanAwsSQSQueues() {
         for (String queueName : context.allSqsQueues()) {
             AwsSqsResource.deleteQueue(queueName);
+            // It is recommended to wait up to 60secs to avoid SQS do some "funky" stuff
```

Comment:
What can actually happen if the waiting is not applied?

Reply:
to be honest, I'm not sure. I wasn't completely sure either whether to keep this or remove it. 
I just added as it was something recommended by the camel-k qe guys in some of their tests. But maybe we can remove for now and put it back in case of some weird SQS behaviour.
wdyt?

Reply:
yes, I would rather avoid such things if they may not be necessary

Reply:
done